oat-sa/extension-lti-test-review
# Feature/TR-157/Use alternative renderers

Related to: https://oat-sa.atlassian.net/browse/TR-157

Requires: 
 - [x] https://github.com/oat-sa/extension-tao-itemqti/pull/1589/files
 - [x] once the companion PR has been merged an update maybe needed on the dependency version for taoQtiItem (the merged version may differ)

Apply the alternative renderer mode in the item runner and remove the read-only overlay which was bringing UX issues.
Also, remove the previous fix made for [TR-117](https://oat-sa.atlassian.net/browse/TR-117) since this change supersedes it.

How to test:
- grab an LTI link for a delivery (the QTI Interactions sample is a good candidate since it contains various type of interactions, including extended text and media)
- open the LTI emulator (http://ltiapps.net/test/tc.php), put the link to the delivery into the launch URL, the key, and the secret, then set the role as `Learner`.
- save and launch the activity into another window
- once the test is finished, copy the deliveryExecution parameter (it will be needed to launch the ltiTestReview)
    <img width="1125" alt="image" src="https://user-images.githubusercontent.com/1500098/100768839-a13efe00-33fb-11eb-93bf-a9d2bdc1ce92.png">
- go back to the LTI emulator
- set the URL as `<HOST>/ltiTestReview/ReviewTool/launch?execution=<DELIVERY_EXECUTION>`, replacing the placeholders by your TAO host and the deliveryExecution parameter copied in a previous step
- tick the checkbox `Display optional parameters`
- write the custom parameters: `show_score=1 show_correct=1`
    <img width="1160" alt="image" src="https://user-images.githubusercontent.com/1500098/100769673-ae102180-33fc-11eb-9e9e-e1f41480ee16.png">
- keep the role as `Learner`.
- save and launch the activity into another window
- the ltiTestReview should start with the delivery and the answer previously given
- the interactions should be reachable, but not modifiable. The media player should work without restriction, and the scrollbars should be actionable too
